packages/devtools-launchpad/webpack.config.devtools.js

```diff
@@ -22,9 +22,7 @@ module.exports = (webpackConfig, envConfig) => {
   }
 
   webpackConfig.devtool = false;
-  webpackConfig.recordsPath = path.join(
-    rootDir, "assets/module-manifest.json"
-  );
+  webpackConfig.recordsPath = path.join(os.tmpdir(), "module-manifest.json");
```

Comment:
Discussed it with @jasonLaster and we surely need to improve the webpack.devtools.config.

We should be able to pass the root dir as a parameter in the env config. We should also only use this file when performing a release (logged #254 about this). 

When I tried it, the module-manifest.json was created in the sourcemap assets folder. Maybe you had problems if you are using linking?

I would say for now simply ignore the module-manifest.json created during the release, and we'll improve this in #254 ?